sdk/identity/identity/src/msal/nodeFlows/msalNodeCommon.ts

```diff
@@ -186,18 +197,32 @@ export abstract class MsalNode extends MsalBaseUtilities implements MsalFlow {
         this.identityClient!.abortRequests(options.correlationId);
       });
     }
-
+    if (options?.enableCae) {
+      this.msalConfig.auth.clientCapabilities = ["cp1"];
```

Comment:
what if `AZURE_IDENTITY_DISABLE_CP1` is set?

Reply:
Thanks for pointing out. I need to remove the support for that. 
The AZURE_IDENTITY_DISABLE_CP1 environment variable is removed since the behavior of the CP1 capability being "always-on" has been changed.